apps/trading/components/rewards-container/active-rewards.tsx

```diff
@@ -365,6 +398,196 @@ const RewardCard = ({
+              <h3 className="flex flex-col gap-1 text-2xl shrink-1 text-center">
+                <span className="font-glitch" data-testid="reward-value">
+                  {rewardAmount}
+                </span>
+
+                <span className="font-alpha">{rewardAsset?.symbol || ''}</span>
+              </h3>
+
+              {/** DISTRIBUTION STRATEGY */}
+              <Tooltip
+                description={t(
+                  DistributionStrategyDescriptionMapping[
+                    DistributionStrategy.DISTRIBUTION_STRATEGY_PRO_RATA
+                  ]
+                )}
+                underline={true}
+              >
+                <span className="text-xs" data-testid="distribution-strategy">
+                  {
+                    DistributionStrategyMapping[
+                      DistributionStrategy.DISTRIBUTION_STRATEGY_PRO_RATA
+                    ]
+                  }
+                </span>
+              </Tooltip>
+            </div>
+
+            {/** DISTRIBUTION DELAY */}
+            <div className="flex flex-col gap-2 items-center text-center">
+              <CardIcon
+                iconName={VegaIconNames.LOCK}
+                tooltip={t(
+                  'Number of epochs after distribution to delay vesting of rewards by'
+                )}
+              />
+              <span
+                className="text-muted text-xs whitespace-nowrap"
+                data-testid="locked-for"
+              >
+                {t('numberEpochs', '{{count}} epochs', {
+                  count: 0,
+                })}
+              </span>
+            </div>
+          </div>
+
+          <span className="border-[0.5px] border-gray-700" />
+          {/** DISPATCH METRIC */}
+          {
+            <span data-testid="dispatch-metric-info">
+              {t('Staking rewards')}
+            </span>
+          }
+          <div className="flex items-center gap-8 flex-wrap">
+            {/** ENDS IN */}
+            {endsIn != null && (
+              <span className="flex flex-col">
+                <span className="text-muted text-xs">{t('Ends in')} </span>
+                <span data-testid="ends-in" data-endsin={endsIn}>
+                  {endsIn >= 0
+                    ? t('numberEpochs', '{{count}} epochs', {
+                        count: endsIn,
+                      })
+                    : t('Ended')}
+                </span>
+              </span>
+            )}
+
+            {/** WINDOW LENGTH */}
+            <span className="flex flex-col">
+              <span className="text-muted text-xs">{t('Assessed over')}</span>
+              <span data-testid="assessed-over">
+                {t('numberEpochs', '{{count}} epochs', {
+                  count: 1,
+                })}
+              </span>
+            </span>
+          </div>
+          {/** DISPATCH METRIC DESCRIPTION */}
+          {
+            <span className="text-muted text-sm h-[3rem]">
```

Comment:
Can we use more semantic elements, this should probably be a p for example

Reply:
updated